--- intl/Makefile.in.orig	Mon Oct  1 12:31:40 2001
+++ intl/Makefile.in	Mon Oct  1 12:32:18 2001
@@ -102,7 +102,7 @@
 	sed -e '/^#/d' -e 's/@''PACKAGE''@/@PACKAGE@/g' $< > t-$@
 	mv t-$@ $@
 
-INCLUDES = -I.. -I. -I$(top_srcdir)/intl
+INCLUDES = -I../include -I. -I$(top_srcdir)/intl
 
 all: all-@USE_INCLUDED_LIBINTL@
 all-yes: libintl.$la libintl.h charset.alias ref-add.sed ref-del.sed
@@ -253,7 +253,7 @@
 
 info dvi:
 
-$(OBJECTS): ../config.h libgnuintl.h
+$(OBJECTS): ../include/config.h libgnuintl.h
 bindtextdom.$lo finddomain.$lo loadmsgcat.$lo: gettextP.h gettext.h loadinfo.h
 dcgettext.$lo: gettextP.h gettext.h hash-string.h loadinfo.h
 
--- po/Makefile.in.in	2001/06/22 08:30:19	1.8
+++ po/Makefile.in.in	2001/06/22 08:32:09	1.9
@@ -23,7 +23,7 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 datadir = @datadir@
-localedir = $(datadir)/locale
+localedir = @LOCALE_DIR@
 gettextsrcdir = $(datadir)/gettext/po
 
 INSTALL = @INSTALL@
@@ -85,6 +85,7 @@
 	$(XGETTEXT) --default-domain=$(PACKAGE) --directory=$(top_srcdir) \
 	  --add-comments --keyword=_ --keyword=N_ \
 	  --files-from=$(srcdir)/POTFILES.in \
+	  --c++ --keyword=plural \
 	&& test ! -f $(PACKAGE).po \
 	   || ( rm -f $(srcdir)/$(PACKAGE).pot \
 		&& mv $(PACKAGE).po $(srcdir)/$(PACKAGE).pot )
